presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -8,13 +8,18 @@ import me.androidbox.component.event.VisitorInfo
 import me.androidbox.domain.authentication.model.Attendee
 import java.time.ZoneId
 import java.time.ZonedDateTime
+import java.util.UUID
 
 data class EventScreenState(
     val listOfPhotoUri: List<String> = mutableStateListOf<String>(),
     val selectedVisitorType: VisitorType = VisitorType.ALL,
     val selectedVisitor: VisitorInfo? = null,
+    val visitorEmail: String = "",
+    val shouldShowVisitorDialog: Boolean = false,
+    val isEmailVerifiedSuccess: Boolean = true,
```

Comment:
Do you need the word "Success" here?
isEmailVerified should be enough :D

Reply:
@MKiperszmid Updated